src/Mailer/Message.php

```diff
@@ -922,11 +922,11 @@ public function getHeaders(array $include = []): array
         $headers['MIME-Version'] = '1.0';
         if ($this->attachments) {
             $headers['Content-Type'] = 'multipart/mixed; boundary="' . (string)$this->boundary . '"';
-        } elseif ($this->emailFormat === static::MESSAGE_BOTH) {
+        } elseif ($this->emailFormat === self::MESSAGE_BOTH) {
```

Comment:
why is this necssary?

Reply:
This isn't necessary for psalm but the class was using a mix of `static` and `self` so I harmonized all to `self`.

Reply:
In modern php shouldnt this all be LSB? It is theoretical, but better for extension, right?

Reply:
Theoretically yes but in practice I don't see any use case for overriding these constants. Regardless I'll change them to static instead.